Update the height and width values in examples

gregsifr opened the pull request:
The dynamic examples were using heights from the generated data instead of the measurement. That is `${rows[virtualRow.index]}px` vs `${virtualRow.size}px`.

The variable example was doing the opposite (using `${virtualRow.size}px`) even though `ref={virtualRow.measureRef}` was never set in the variable example.

Also update the condition in infinite scroll (#161)

TkDodo commented:
to try out the changes to the examples on this PR, codesandbox preview builds would be ideal, like the ones we have on react-query. I'm happy to PR the config changes, but we'd need @tannerlinsley to enable the bot as far as I remember, given that he's the owner :)

gregsifr commented:
Updated infinite scroll example:  https://codesandbox.io/s/ecstatic-tharp-u3llm
Updated variable height example: https://codesandbox.io/s/fast-frog-tc4l8?file=/src/index.js
Updated dynamic height example: https://codesandbox.io/s/distracted-tdd-o7wh7?file=/src/index.js

tannerlinsley commented:
:tada: This PR is included in version 2.10.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/react-virtual/v/2.10.0)
- [GitHub release](https://github.com/tannerlinsley/react-virtual/releases/tag/v2.10.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

tannerlinsley commented:
:tada: This PR is included in version 2.9.1 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/react-virtual/v/2.9.1)
- [GitHub release](https://github.com/tannerlinsley/react-virtual/releases/tag/v2.9.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
